app/Http/Middleware/RoomAuthenticate.php

```diff
@@ -55,21 +58,37 @@ public function handle($request, Closure $next, $allowUnAuthenticated = false)
 
         // user is not authenticated and room is not allowed for guests
         if (! $room->getRoomSetting('allow_guests') && ! $authenticated && ! Auth::user()) {
+            Log::notice('Room guest access failed for room {room}', ['room' => $room->getLogLabel()]);
+
             abort(403, 'guests_not_allowed');
         }
 
-        // if room has not access code
+        // if room has no access code
         if ($room->access_code == null) {
             $authenticated = true;
         }
 
         // request provided access code
         if ($request->headers->has('Access-Code')) {
+
+            // Key used to rate limit access code attempts (per room)
```

Comment:
```suggestion
            // Key used to rate limit access code attempts
```

Comment needs to be adjusted (df42d2a962a11dabdda1dac0acdaf120528f1e63)